src/envo/envo-edit.owl

```diff
@@ -40800,6 +40811,94 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03500019> <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03500019> "2024-01-19T18:34:02Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03500019> "lead solder"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500019> <http://purl.obolibrary.org/obo/ENVO_03500018>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03500020> (non-lead solder)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03500020> "A solder which is composed without lead.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03500020> <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03500020> "2024-01-19T18:34:10Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03500020> "non-lead solder"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500020> <http://purl.obolibrary.org/obo/ENVO_03500018>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03500021> (gasoline vapor)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03500021> "A fume produced by liquid gasoline, containing aerosolized components of the gasoline.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03500021> <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03500021> "2024-01-19T18:39:48Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03500021> "gasoline vapor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500021> <http://purl.obolibrary.org/obo/ENVO_02000112>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03500022> (noise)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.wikidata.org/wiki/Q179448") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03500022> "An energy transfer process that moves vibrations through a medium, such as air or water, that is received and perceived by an organism and considered unpleasant, loud, or disruptive.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_03500022> <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_03500022> "2024-01-19T18:52:13Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03500022> "noise"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500022> <http://purl.obolibrary.org/obo/ENVO_01001852>)
```

Comment:
at some point we need to reconcile the wave energy and energy transfer parts of ENVO... but that is for another time

Reply:
ok